Docs/source/running_cpp/parameters.rst

```diff
@@ -1676,6 +1676,19 @@ Reduced Diagnostics
         Note that the fields are averaged on the cell centers before their maximum values are
         computed.
 
+    * ``RhoMaximum``
+        This type computes the maximum and minimum values of the total charge density as well as
+        the maximum absolute value of the charge density of each charged species.
+        Please be aware that measuring maximum charge densities might be very noisy in PIC simulations.
+
+        The output columns are
+        the maximum value of the :math:`rho` field,
+        the minimum value of the :math:`rho` field,
+        the maximum value of the absolute :math:`|rho|` field of each charged species.
```

Comment:
I wonder if it would be possible and useful to also write out where this maximum/minimum was found (spatially)?

Reply:
Interesting idea. I don't know how easy that would be to do with AMReX reduction operations.

Regarding the usefulness: I guess it depends on the specific scenario. For us it wouldn't be as useful as the absolute density values (we already have a rough idea of where the density peak is and don't really need that information as frequently) but it would still be quite interesting.

So in general I think it could be a good feature to have (also for the FieldMaximum reduced diag), provided that it's not too hard to implement. Do you think we should try to do this? If so, here or in a follow-up PR?

Reply:
Thanks for the details. Yes, can be a follow-up PR! I was curious, e.g. to follow e-/p+ fronts or so.